static/app/components/u2f/u2finterface.tsx

```diff
@@ -111,18 +126,49 @@ class U2fInterface extends React.Component<Props, State> {
       });
   }
 
+  webAuthnSignIn(authRequest) {
+    const credentials = [] as any;
+
+    authRequest.forEach(device => {
+      credentials.push({
+        id: base64urlToBuffer(device.keyHandle),
+        type: 'public-key',
+        transports: ['usb', 'ble', 'nfc'],
+      });
+    });
+
+    const publicKeyCredentialRequestOptions = {
+      challenge: base64urlToBuffer(authRequest[0].challenge),
```

Comment:
here it seems we discard the rest of the requests potentially in the array? whats going on with that?